f5/bigip/tm/sys/management_route.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+#
+# Copyright 2016 F5 Networks Inc.
```

Comment:
heh, I suppose I should read comment fields I copy over from other files more carefully...I can clean up here or I can submit a PR to update all source files for copyright info.